bears/general/RegexLintBear.py

```diff
@@ -0,0 +1,54 @@
+import re
+
+from queue import Queue
+from sarge import run, Capture
+from contextlib import suppress
+
+from bears.general.AnnotationBear import AnnotationBear
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Result import Result
+from coalib.settings.Section import Section
+from coalib.settings.Setting import Setting
+from coalib.testing.LocalBearTestHelper import execute_bear
+
+from dependency_management.requirements.PipRequirement import PipRequirement
+
+
+class RegexLintBear(LocalBear):
+    LANGUAGES = {'All'}
+    REQUIREMENTS = {PipRequirement('regexlint', '1.6')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting'}
+
+    def run(self, filename, file, language: str):
+        """
+        Bear for linting regex through regexlint.
+
+        :param language:
+            The language of the files, must be supported by coala.
```

Comment:
We use common description for language, 'The programming language of the file(s).'.